Sasquatch/Config/Sasquatch.xcconfig

```diff
@@ -1,7 +1,7 @@
 #include "../../Config/App.xcconfig"
 
 PRE_XCODE_10_1_ARCHS = armv7 armv7s arm64
-ARCHS[sdk=iphoneos*] = $(PRE_XCODE_10_1_ARCHS) arm64e
+ARCHS[sdk=iphoneos*] = $(PRE_XCODE_10_1_ARCHS)
```

Comment:
I would set arm64e on Debug builds so that we can still test it.

Reply:
Can we clean up PRE_XCODE_10_1_ARCHS? We dropped Xcode 8/9 and officially use 10.1 for build. We should be able to remove this. @clpolet any concerns?

Reply:
We need to keep it for Xcode 10.0. But you can remove all the lines referencing older Xcode versions so lines containing:

```
ARCHS[sdk=iphoneos9.*] = $(PRE_XCODE_10_1_ARCHS)
ARCHS[sdk=iphoneos10.*] = $(PRE_XCODE_10_1_ARCHS)
ARCHS[sdk=iphoneos11.*] = $(PRE_XCODE_10_1_ARCHS)
```

Reply:
But can be done in a separate PR. We will also have to get rid of the `PRE_XCODE_10_1_WATCH_ARCHS`, arm64_32 came with Xcode 10.